interactive_trainer.py

```diff
@@ -461,7 +465,14 @@ def get_sample(self, folder, sample_name=None):
         }
         return img, None, info
 
-    def push_sample(self, sample_name, geojson_annotation, target_folder="train"):
+    def push_sample_async(self, paras):
+        self.queue.sync_q.put({"type": "push_sample", "paras": paras})
```

Comment:
I think it's better to keep the expanded function signature for `push_sample` and `push_sample_async`.
```suggestion
def push_sample_async(self, *args, **kwargs):
        self.queue.sync_q.put({"type": "push_sample", "args": args, "kwargs": kwargs})
```

Reply:
agree. I just modified accordingly